script.js

```diff
@@ -26,112 +29,333 @@ function closeMenu() {
   menuOpen = false;
 }
 
-const intro = [
-  {
-    cls: "intro-text",
-    h1: "Hey There",
-    h1h: "I'm Evans",
-    h2: "I'm a software Developer",
-    para: "I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.",
-    cls1: "socials",
-    icns: [
-      "fa-brands fa-github",
-      "fa-brands fa-linkedin-in",
-      "fa-brands fa-twitter",
-      "fa-solid fa-m",
-    ],
-  },
-];
-
-const section = ["intro", "portfolio", "contact", "about"];
-const footer = ["contact", "copyright"];
-
-const title = [
-  {
-    cls: "title",
-    h2: "My Recent Works",
-  },
-];
-
-const plchdr = [
+const worksInfo = [
   {
-    cls: "img",
-    src: "../images/plc-img.jpg",
-    alt: "A woman doing yoga",
-    cls1: "text",
-    h2: "Multi-Post-Stories",
-    para: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
-    list: ["css", "html", "bootstrap", "ruby"],
-    btn: "See Project",
-  },
-];
-
-const cards = [
-  {
-    cls: [
+    classes: [
+      "plchdr",
+      "img",
+      "text",
       "card cd1",
       "card cd2",
       "card cd3",
       "card cd4",
       "card cd5",
       "card cd6",
+      "seeDemo",
+      "seeDemo2",
     ],
-    h3: "Professional Art Printing Data",
-    para: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
-    list: ["css", "html", "bootstrap"],
+    heading: ["Professional Art Printing Data", "Multi-Post-Stories"],
+    plchdrImage: "../images/plc-img.jpg",
+    alt: "a woman doing yoga",

```

Comment:
- [ ] This image is not appearing
![Screen Shot 2022-11-25 at 12 50 59](https://user-images.githubusercontent.com/56886064/203967883-f1348edc-1a62-495d-a858-02fbebd2995d.png)